### ISSUE TITLE
Implement mergeOptions command

### ISSUE THREAD

**Issue by Powersource** [staltz/cycle-native-navigation-web#1]
Used by the sidebar in manyverse

Some related links I found  
https://github.com/staltz/cycle-native-navigation-web/blob/main/src/run.ts  
https://github.com/staltz/cycle-native-navigation/blob/master/src/global.ts  
https://wix.github.io/react-native-navigation/docs/sideMenu/

**Comment by staltz** [comment]
One note, though: mv desktop will have a "frame", not a sideBar, but some of sideBar options will control the frame. At least, whether it's visible or not. Maybe other properties. In mv desktop we don't need to support the `drawer` at all, we will have a separate "screen" that will serve as the frame. We could even name it just `frame`. The hamburger button on the top left corner would not exist on desktop. But all of this is stuff that still needs UI design. Maybe we can just design it as we go.

**Comment by Powersource** [comment]
Ah you mean it would just be attached to the left side of the screen always, like patchwork? That'd make things easier

**Comment by staltz** [comment]
Yes! The UI design is pending so in practice we'll probably just implement something, and then tweak it many times until it's right. David might have thoughts to help out with that. Anyway, you can forget `drawer` entirely for mv desktop.
